src/base/base.scss

```diff
@@ -77,10 +77,63 @@ button {
 }
 
 details {
-  summary { cursor: pointer; }
+  summary {
+    cursor: pointer;
+  }
 
   &:not([open]) {
     // Set details content hidden by default for browsers that don't do this
-    > *:not(summary) { display: none !important; }
+    > *:not(summary) {
+      display: none !important;
+    }
+  }
+}
+
+// global focus styles
+
+a,
+button,
+[role='button'],
+input,
+textarea,
+select {
+  // fallback :focus state
+  &:focus {
+    @include focusOutline;
```

Comment:
I think this is causing issues with stacking context in the [forms docs](https://primer-css-b32u0b8ly-primer.vercel.app/css/components/forms).

https://user-images.githubusercontent.com/2313998/149568818-a49f8d13-de96-4e14-aa5f-00de14fada6d.mp4

Reply:
Thank you @mperrotti! Looks like you found a bug in an older Chrome (but a stable version so critical that we cover it). Removing `position: relative;` and `z-index` seemed to fix the issue 🤞